examples/bone_finetuning/scripts/run_full_finetune.sh

```diff
@@ -0,0 +1,25 @@
+BASE_MODEL="/home/rwkv/JL/model/qwen2-1.5b"
+OUTPUT_PATH="output/FullFT-Llama-3-8B"
+DATA_PATH="/home/rwkv/JL/model/MetaMathQA"
+
+# batch size = per_device_train_batch_size * gradient_accumulation_steps * num_gpus = 128
+deepspeed --master_port=16971 --include=localhost:0 pissa.py \
```

Comment:
There is no `pissa.py` script. Also, why use PiSSA when this is supposed to be for full finetuning? Honestly, I think it's enough to just have the Bone script, no need for full finetuning, LoRA, and PiSSA.